src/local.c

```diff
@@ -201,9 +201,21 @@ update_localdb(int verbose)
     }
 
     if (rename(tmp, outhome)) {
-        fprintf(stderr, "Error: Could Not Rename: %s to %s\n", tmp, outhome);
-        rm(outpath);
-        return 1;
+        if (errno == EXDEV) {
+            /* tmp and outhome are on different file systems, do full copy
+             * and rm instead.
+             */
+            if(copytree(tmp, outhome)) {
+                fprintf(stderr, "Error: Could Not Move: %s to %s\n",
+                        tmp, outhome);
+                return 1;
+            }
+            /*rm(tmp);*/
```

Comment:
Yeah, I thought I had to remove the directory for the move but the directory gets removed in the next if block with its parent outpath. I don't know why I commented it out, but either way it's useless and I'm going to remove it.